internal/saucecloud/imagerunner.go

```diff
@@ -391,51 +397,56 @@ func (r *ImgRunner) PollRun(ctx context.Context, id string, lastStatus string) (
 	}
 }
 
-func (r *ImgRunner) DownloadArtifacts(runnerID, suiteName, status string, passed bool) {
+// DownloadArtifact downloads a zipped archive of artifacts and extracts the required files.
+func (r *ImgRunner) DownloadArtifacts(runnerID, suiteName, status string, passed bool) []string {
 	if r.Async ||
 		runnerID == "" ||
 		status == imagerunner.StateCancelled ||
 		!r.Project.Artifacts.Download.When.IsNow(passed) {
-		return
+		return nil
 	}
 
 	dir, err := config.GetSuiteArtifactFolder(suiteName, r.Project.Artifacts.Download)
 	if err != nil {
 		log.Err(err).Msg("Unable to create artifacts folder.")
-		return
+		return nil
 	}
 
 	log.Info().Msg("Downloading artifacts archive")
 	reader, err := r.RunnerService.DownloadArtifacts(r.ctx, runnerID)
 	if err != nil {
 		log.Err(err).Str("suite", suiteName).Msg("Failed to fetch artifacts.")
-		return
+		return nil
 	}
 	defer reader.Close()
 
 	fileName, err := fileio.CreateTemp(reader)
 	if err != nil {
 		log.Err(err).Str("suite", suiteName).Msg("Failed to download artifacts content.")
-		return
+		return nil
 	}
 	defer os.Remove(fileName)
 
 	zf, err := zip.OpenReader(fileName)
 	if err != nil {
-		log.Error().Msgf("Unable to open zip file %s: %s", fileName, err)
-		return
+		log.Err(err).Msgf("Unable to open zip file '%s'", fileName)
```

Comment:
`%q` does the quoting:
```suggestion
		log.Err(err).Msgf("Unable to open zip file %q", fileName)
```